locale/en/knowledge/command-line/how-to-get-colors-on-the-command-line.md

```diff
@@ -39,19 +39,19 @@ console.log(stringThree.grey.bold);
 
 There are several things to take note of here - first, the string object has been prototyped, so any color may be added simply by adding the property to the string!  It works on string literals, template literals and on variables, as shown at the top of the example above.
 
-Notice, also, from the second pair of `console.log` statements, that once set, a color value persists as part of the string.  This is because under the hood, the proper ANSI color tags have been prepended and appended as necessary - anywhere the string gets passed where ANSI color codes are also supported, the color will remain.
+Notice, also, from the second pair of `console.log` statements, that once set, a color value persists as part of the string. This is because under the hood, the proper ANSI color tags have been prepended and appended as necessary - anywhere the string gets passed where ANSI color codes are also supported, the color will remain.
 
-The last pair of `console.log` statements are probably the most important.  Because of the way `colors.js` and ANSI color codes work, if more than one color property is set on a string, **only the first color property to be set on the string takes effect.**  This is because the colors function as 'state shifts' rather than as tags.
+The last pair of `console.log` statements are probably the most important. Because of the way `colors.js` and ANSI color codes work, if more than one color property is set on a string, **only the first color property to be set on the string takes effect.**  This is because the colors function as 'state shifts' rather than as tags.
```

Comment:
```suggestion
The last pair of `console.log` statements are probably the most important. Because of the way `colors.js` and ANSI color codes work, if more than one color property is set on a string, **only the first color property to be set on the string takes effect.** This is because the colors function as 'state shifts' rather than as tags.
```